### PR TITLE
feat: 'setup codestyle for team' implementation

### PR DESCRIPTION
**Pull request by vzanimonets**
First of all you should to download the ESLint and Prettier extensions for VSCode

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,6 +19,14 @@
     "react-helmet": "^5.2.1"
   },
   "devDependencies": {
```
Comment (rss-maksim) [inline package.json:25]: may be it's necessary to add husky for pre-commit hook? 

like this 
```"lint-staged": {
    "*.{js,jsx}": [
      "npx eslint -c eslint.json",
      "git add"
    ]
  }
```
```
"husky": {
    "hooks": {
      "pre-commit": "npx lint-staged"
    }
  }
```

Comment (spirithntr) [reply package.json:25]: So what you are trying to say is actually: 
```
"husky": {    
  "hooks": {
      "pre-commit": "npx lint-staged"
    }
}
```
where husky is npm package that should be installed.
Sounds reasonable to me.

Comment (rss-maksim) [reply package.json:25]: Yes)

Comment (vzanimonets) [reply package.json:25]: I install packages hasky & lint-stages, then add following code to psckage.json:
```
"lint-staged": {
    "src/**/*.{js,jsx}": [
      "eslint . --fix",
      "git add"
    ]
  },
  "husky": {
    "hooks": {
      "pre-commit": "lint-staged"
    }
  },
``` 
but how it should work?)

Comment (rss-maksim) [reply package.json:25]: it should be triggered hook after your commiting

**Review by rss-maksim — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (vzanimonets) [reply package.json:25]: I updated commit. Can you check it?

**Comment by vadzim-sasonkin** [comment]
@vzanimonets 
We need to resolve conflicts in `package-lock.json` and merge this branch

File: package-lock.json [hunk 2]
```diff
@@ -8750,10 +8993,124 @@
...
+          "resolved": "https://registry.npmjs.org/supports-color/-/supports-color-7.1.0.tgz",
+          "integrity": "sha512-oRSIpR8pxT1Wr2FquTNnGet79b3BWljqOuoW/h4oBhxJ/HUbX5nX6JSruTkvXDCFMwDPvsaTTbvMLKZWSy0R5g==",
+          "dev": true,
+          "requires": {
+            "has-flag": "^4.0.0"
+          }
+        }
+      }
+
     "hyphenate-style-name": {
```
Comment (vadzim-sasonkin) [inline package-lock.json:9109]: @vzanimonets should be a comma before this key

**Pull request merged**